processing/src/test/java/org/apache/druid/segment/data/IncrementalIndexTest.java

```diff
@@ -268,7 +217,7 @@ private static MapBasedInputRow getLongRow(long timestamp, int dimensionCount)
   public void testCaseSensitivity() throws Exception
   {
     long timestamp = System.currentTimeMillis();
-    IncrementalIndex index = closerRule.closeLater(indexCreator.createIndex(DEFAULT_AGGREGATOR_FACTORIES));
+    IncrementalIndex<?> index = indexCreator.createIndex((Object) DEFAULT_AGGREGATOR_FACTORIES);
```

Comment:
all changes from here are due to the generic type?

Reply:
Yes. To allow testing any new incremental index.

Reply:
Thanks for addressing the questions and issues
LGTM